pyedflib/tests/test_highlevel.py

```diff
@@ -39,7 +39,7 @@ def tearDownClass(cls):
                 os.remove(os.path.join(data_dir, file))
             except Exception as e:
                 print(e)
-                
+
```

Comment:
Apologies about the extra blank lines which were automatically added by my VSCode upon saving the file 😬

Reply:
no problem